completion/available/projects.completion.bash

```diff
@@ -11,7 +12,7 @@ _pj() {
 	shift
 	[ "$1" == "open" ] && shift
 
-	local cur prev words cword
+	local cur
```

Comment:
`prev`, `words`, and `cword` will be set by `_init_completion`, so they need to be localized.

Reply:
I'lll check again. thanks!